### ISSUE TITLE
Scroll bar doesn't capture mouse

### ISSUE THREAD

**Issue by Timmmm** [PistonDevelopers/conrod#1245]
I just tried the examples, and a pretty obvious flaw from the start is that the scroll bar (which is initially invisible - is that intentional?) doesn't capture the mouse. So if you start dragging and then continue dragging with the mouse outside the Conrod window, scrolling will stop.

**Comment by mitchmindtree** [comment]
Thanks for the issue!

Yes, this is basically the same issue as #788 and similarly could be fixed with the same solution as mentioned [in this comment](https://github.com/PistonDevelopers/conrod/issues/788#issuecomment-440551019).

**Comment by mitchmindtree** [comment]
Ahh sorry, the way you describe scrolling stopping once the mouse leaves the scroll surface does sound like a different bug.

Would you mind sharing what OS you are on and which examples in particular you know this problem happens for?

I have a feeling this might be a difference in the way in which different OSes provide mouse events via winit - e.g. some OSes continue to provide mouse moved events if the window is still in focus when the mouse is outside the window, whereas others do not provide mouse moved events outside the window, focused or not. If this is the case, I might make an issue at winit to try and make sure the behaviour is consistent before fixing any logic if necessary. I am pretty sure that on my linux machine and the last time I used a mac, scrolling would continue while moving the cursor outside the bounds of the window. E.g. this is what happens for me on linux (sorry about the potato quality gif, github won't let me upload webm):

![conrod_scroll](https://user-images.githubusercontent.com/4587373/51654933-67766280-1fee-11e9-8d18-5de87ea2791d.gif)

**Comment by Timmmm** [comment]
Yeah that's exactly what I mean - I'm on MacOS 10.13.6 and scrolling stops when the mouse leaves the window (unlike in your video). I believe I was using the `all_winit_glium` example.

**Comment by alvinhochun** [comment]
A fix landed in `winit`: https://github.com/rust-windowing/winit/pull/1607 but it hasn't been released yet.
